.github/workflows/tests.yml

```diff
@@ -138,7 +138,7 @@ jobs:
 
   prepare:
     name: "🔍 Check source preparation"
```

Comment:
You probably need to install the build deps here, now that this step also generates the list of images which compiles `images`.

So probably `sudo apt install -y libgpgme-dev libbtrfs-dev libdevmapper-dev`

Reply:
Oops, yes.  Absolutely right.